extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -47,7 +47,8 @@ private function checkArguments( array $args, $modelData, &$errorMessage ) {
 			$args['mostrecent'] !== 'false'
 		) {
 			$errorMessage = wfMessage( 'embeddable-discussions-parameter-error', 'mostrecent',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "embeddable-discussions-parameter-error" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)